config/locales/views/moderations/en.yml

```
@@ -53,11 +53,11 @@ en:
             Expert: Expert
             Mid-level: Mid-level
             Novice: Novice
-        admin:
-          heading: Admin actions
-          heading2: Higher permission post options
-          subtitle: Higher permission post options
-          aria_label: Open admin actions
+        moderator:
+          heading: Moderator actions
+          heading2: "Moderating actions to %{user}"
+          subtitle: "Moderating actions to %{user}"
```

```suggestion
          heading2: "Moderating actions for %{user}"
          subtitle: "Moderating actions for %{user}"
```

? IDK it doesn't really scan for me (consider this non-blocking, just wanted to flag up as it seems a bit off to me)

---

yeah... I believe this is how it was on the Figma. Wasn't sure if there was a reason it should read as such. I agree with you though and will commit this suggestion!